wordpress-mobile/WordPress-iOS
# Issue/8686 async posting

Fixes #8686 

This is still a work in progress, but it already allows to leave the editor, start an async upload when media is still uploading and then updates the post content in the background when the media uploads are finished.

Still need to do handle the error scenarios, but if you want to give it an initial look @frosty 

To test:
 - Start an new post
 - Add multiple media to it
 - Tap on the ellipsis and chose Async Upload
 - Check the console for media progress updates
 - When media is finished go back to the post editor and see if it was updated correctly.
